src/NWCClient.ts

```diff
@@ -334,11 +335,21 @@ export class NWCClient {
     return getPublicKey(hexToBytes(this.secret));
   }
 
-  get supportedVersion(): string {
-    if (!this.version) {
-      throw new Error("Missing version");
+  get supportedEncryptionOrVersion(): string {
+    if (!this.encryptionOrVersion) {
+      throw new Error("Missing encryption or version");
     }
-    return this.version;
+    return this.encryptionOrVersion;
+  }
+
+  isUsingVersioning(encryptionOrVersion = this.encryptionOrVersion): boolean {
```

Comment:
code like this will be unfortunately hard to remove, let's huddle together about it next week